### PR TITLE
Deprecate this repository

### PR DESCRIPTION
**Pull request by elfosardo**
After the merge with ironic-image, we should deprecate the repository
and stop accepting any PR.

This should happen after https://github.com/openshift/ironic-image/pull/179

### CODE REVIEW

**Comment by elfosardo** [comment]
this probably needs https://github.com/openshift/release/pull/19156

**Comment by elfosardo** [comment]
we can actually override the jobs so we just need https://github.com/openshift/ironic-image/pull/179 merged before merging this one

**Comment by derekhiggins** [comment]
> tide Pending — Not mergeable. Needs lgtm label.

tide doesn't mention the hold label... I was going to /lgtm so you could /unhold when ready but I'm slightly worried it will merge it...

**Comment by elfosardo** [comment]
> > tide Pending — Not mergeable. Needs lgtm label.
> 
> tide doesn't mention the hold label... I was going to /lgtm so you could /unhold when ready but I'm slightly worried it will merge it...

@derekhiggins mmm weird, let's not lgtm it until https://github.com/openshift/ironic-image/pull/179 has merged

**Review by iurygregory — APPROVED** [review]
/lgtm

**Pull request merged**
